src/3D/LandIslandInterface.h

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2018-2023 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *******************************************************************************/
+
+#pragma once
+#include <filesystem>
+
+#include <LNDFile.h>
+#include <glm/mat4x4.hpp>
+
+using LNDCell = openblack::lnd::LNDCell;
+
+namespace openblack
+{
+class LandBlock;
+namespace graphics
+{
+class FrameBuffer;
+class Texture2D;
+} // namespace graphics
+class LandIslandInterface
+{
+public:
+	virtual void LoadFromFile(const std::filesystem::path& path) = 0;
+	[[nodiscard]] virtual float GetHeightAt(glm::vec2) const = 0;
+	[[nodiscard]] virtual const LandBlock* GetBlock(const glm::u8vec2& coordinates) const = 0;
```

Comment:
Unused so I wouldn't put it in the interface
```suggestion
```

Reply:
Fixed on next commit